Use Time instead of DateTime

emeryamiller opened the pull request:
- Active record works better with Time, as opposed to DateTime objects,
  see issue #668 for more details.

Note: I updated the code, but the test cases kept working despite the change.  I updated the revokable test cases anyway, to match the change above, and the test cases are still passing.

tute commented:
That is great, thank you! Would please update the NEWS file too? And, if you can add more context (your Note in the PR description, the linked gist in the issue, and the bug itself) to the commit message this will be world class. Thank you! :)

emeryamiller commented:
Ok, I've updated.  Just one question, I updated the News file by adding an 3.0.0 (rc2) version, I'm not sure if that's right, let me know if it's not.

tute commented:
Thanks so much!
